pkg/storj/node.go

```diff
@@ -74,6 +74,18 @@ func (id NodeID) String() string {
 // Bytes returns raw bytes of the id
 func (id NodeID) Bytes() []byte { return id[:] }
 
+// Less returns whether id is smaller than b
```

Comment:
maybe add into the comment what it means for an ID to be less than another?

Reply:
fixed